app/models/task.rb

```diff
@@ -111,15 +111,15 @@ def author?(user)
   end
 
   def showable_by?(user)
-    user.nil? ? false : user.can?(:show, self)
+    user.nil? ? false : Pundit.policy(user, self).show?
   end
 
   def updateable_by?(user)
-    user.nil? ? false : user.can?(:update, self)
+    user.nil? ? false : Pundit.policy(user, self).update?
   end
 
   def destroyable_by?(user)
-    user.nil? ? false : user.can?(:destroy, self)
+    user.nil? ? false : Pundit.policy(user, self).destroy?
   end
```

Comment:
In the next slice (after we merged this PR), we should probably convert those methods to real policies as well.

Reply:
I created a ticket so we remember this: #1142